features/facebookImport/src/model/analyses/advertising-value-analysis.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import RootAnalysis from "./root-analysis.js";
+import i18n from "../../i18n.js";
+
+import AdvertisingValueMiniStory from "../../components/advertisingValueMiniStory/advertisingValueMiniStory.jsx";
+
+export default class AdvertisingValueAnalysis extends RootAnalysis {
+    get title() {
+        return i18n.t("advertisingValueMiniStory:title");
+    }
+
+    async analyze({ facebookAccount }) {
+        const randomAdInterests = [];
+        const numberInterests = facebookAccount.adInterests.length;
+        if (numberInterests > 0)
+            for (let i = 0; i < 3; i++) {
```

Comment:
Would it be possible to adapt this depending on the screen size?

Reply:
It's extremely unlikely interests number less than 3, but anyway would be best to check.

Reply:
In general yes, but it's too much hassle for right now

Reply:
Maybe pick the max between 3 and numberInterests?